.github/workflows/e2e_lighthouse_performance_acceptance_tests.yml

```diff
@@ -12,60 +12,10 @@ on:
       - release-*
 
 jobs:
-  build:
-    runs-on:  ${{ matrix.os }}
-    strategy:
-      matrix:
-        os: [ubuntu-22.04]
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-python@v4
-        with:
-          python-version: '3.8.15'
-          architecture: 'x64'
-          cache: 'pip'
-          cache-dependency-path: |
-            requirements.in
-            requirements.txt
-            requirements_dev.in
-            requirements_dev.txt
-      - uses: ./.github/actions/merge
-      - name: Cache node modules and third_party/static
-        uses: actions/cache@v3
-        env:
-          cache-name: cache-node-modules
-        with:
-          path: |
-            /home/runner/work/oppia/yarn_cache
-            /home/runner/work/oppia/oppia/third_party/static
-          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('yarn.lock', 'dependencies.json') }}
-          restore-keys: |
-            ${{ runner.os }}-build-${{ env.cache-name }}-
-            ${{ runner.os }}-build-
-            ${{ runner.os }}-
-      - uses: ./.github/actions/install-oppia-dependencies
-      - name: Install chrome
-        run: python -m scripts.install_chrome_for_ci
-      - name: Install ffmpeg
-        run: sudo apt install ffmpeg
-      - name: Build Webpack
-        run: python -m scripts.build --prod_env
-      - name: Zip build files
-        # We avoid using ../ or absolute paths because unzip treats these as
-        # security issues and will refuse to follow them.
-        run: |
-          zip -rqy build_files.zip oppia/third_party oppia_tools oppia/build oppia/webpack_bundles oppia/proto_files oppia/app.yaml oppia/assets/hashes.json oppia/proto_files oppia/extensions/classifiers/proto/* oppia/backend_prod_files oppia/dist
-        working-directory: /home/runner/work/oppia
-      - name: Upload build files artifact
-        uses: actions/upload-artifact@v3
-        with:
-          name: build_files
-          path: /home/runner/work/oppia/build_files.zip
-          retention-days: 1
   e2e_test:
-    needs: build
     runs-on:  ${{ matrix.os }}
     strategy:
+      fail-fast: false
```

Comment:
Why turn this off? We enabled it to avoid saturating the CI queue with e2e jobs on PRs that had already failed CI checks. Ditto below

Reply:
Fixed